### PR TITLE
KAFKA-15210: Mention vote should be open for at atleast 72 hours in the release script

### PR DESCRIPTION
**Pull request by tinaselenge**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: release.py [hunk 1]
```diff
@@ -721,7 +721,7 @@ def select_gpg_key():
 Release notes for the %(release_version)s release:
 https://home.apache.org/~%(apache_id)s/kafka-%(rc_tag)s/RELEASE_NOTES.html
 
-*** Please download, test and vote by <VOTING DEADLINE, e.g. Monday, March 28, 9am PT>
+*** Please download, test and vote by <VOTING DEADLINE, e.g. Monday, March 28, 9am PT, which should be at least 72 hours from now.>
```
Comment (divijvaidya) [inline release.py]: can you please add the reason for 72 hours here as well? You can provide a link to https://www.apache.org/legal/release-policy.html#release-approval.

Comment (tinaselenge) [reply release.py]: Thank you for reviewing the change @divijvaidya. I added a little more details as suggested. Please let me know if not sufficient.

File: release.py [hunk 2]
```diff
@@ -722,6 +722,7 @@ def select_gpg_key():
 https://home.apache.org/~%(apache_id)s/kafka-%(rc_tag)s/RELEASE_NOTES.html
 
 *** Please download, test and vote by <VOTING DEADLINE, e.g. Monday, March 28, 9am PT>
+<THE RELEASE POLICY ( https://www.apache.org/legal/release-policy.html#release-approval) REQUIRES VOTES TO BE OPEN FOR MINIMUM OF 3 DAYS THEREFORE VOTING DEADLINE SHOULD BE AT LEAST 72 HOURS FROM THE TIME THIS EMAIL IS SENT.
```
Comment (divijvaidya) [inline release.py]: closing `>` is missing, otherwise looks good

**Review by divijvaidya — APPROVED** [review]
Thank you for working on this!

**Comment by tinaselenge** [comment]
@divijvaidya thank you for reviewing the PR!

**Pull request merged**